airtime_mvc/application/forms/LiveStreamingPreferences.php

```diff
@@ -7,6 +7,7 @@ public function init()
     {
         $CC_CONFIG = Config::getConfig();
         $isDemo = isset($CC_CONFIG['demo']) && $CC_CONFIG['demo'] == 1;
+        $isStreamConfigable = Application_Model_Preference::GetEnableStreamConf() == "true";
```

Comment:
This was used in 2.5 to do something with the demo mode, it's currently unused and this line can be removed.